packages/ide-extension/package.json

```diff
@@ -44,6 +44,25 @@
         "onCommand:sap.ux.knowledgeHub.openKnowledgeHub"
     ],
     "contributes": {
+        "configuration": {
+            "id": "sap.ux.knowledgeHub",
+            "type": "object",
+            "title": "%configuration.title%",
+            "properties": {
+                "sap.ux.knowledgeHub.openInNewTab": {
```

Comment:
Why do you need this? It doesn't seem to be related to the telemetry user story.